runtime/pom.xml

```diff
@@ -22,6 +22,10 @@
       <groupId>com.asyncapi</groupId>
       <artifactId>asyncapi-core</artifactId>
     </dependency> 
+    <dependency>
```

Comment:
This is needed to avoid a problem with the version of lombok that is referenced from AsyncAPI dependency.